File: abc/abc340/a.cs
// https://atcoder.jp/contests/abc340/submissions/50216148

using static System.Console;

public class A
{
  static void Main()
  {
    string[] t = ReadLine().Split();
    int a = int.Parse(t[0]);
    int b = int.Parse(t[1]);
    int d = int.Parse(t[2]);

    for(int i = a; i <= b; i += d)
    {
      Write($"{i} ");
    }
  }
}

File: abc/abc340/a.cpp
// https://atcoder.jp/contests/abc340/submissions/50216136

#include <bits/stdc++.h>
using namespace std;

int main() {
  int a, b, d;
  cin >> a >> b >> d;

  for(int i = a; i <= b; i += d) {
    printf("%d ", i);
  }
}
